pkg/domain/infra/abi/play.go

```diff
@@ -453,11 +453,14 @@ func kubeContainerToCreateConfig(ctx context.Context, containerYAML v1.Container
 
 	containerConfig.Command = []string{}
 	if imageData != nil && imageData.Config != nil {
-		containerConfig.Command = append(containerConfig.Command, imageData.Config.Entrypoint...)
+		containerConfig.Command = imageData.Config.Entrypoint
 	}
 	if len(containerYAML.Command) != 0 {
-		containerConfig.Command = append(containerConfig.Command, containerYAML.Command...)
-	} else if imageData != nil && imageData.Config != nil {
+		containerConfig.Command = containerYAML.Command
```

Comment:
should this still be an append, or do we want to take the yaml over the config data?  I think this is correct as is, but just wanted to double check.

Reply:
pod's command should override the image's Entrypoint
pod's args should override the image's Command

Reply:
could see this doc
https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes

| Description	| Docker field name	| Kubernetes field name	|
| :----- | :----- | :----- |
|The command run by the container	| Entrypoint	| command	|
|The arguments passed to the command	| Cmd	| args |